src/html/fragments/fonts.html

```diff
@@ -32,6 +32,18 @@
         font-size: 17.4px;
     }
 
+{% elif locale == "ko-KR" %}
+
+    @font-face {
+        font-family: "Noto Sans Korean";
+        src: url(./fonts/NotoSans-Regular.woff) format("WOFF");
```

Comment:
let's make sure to rename this to include the "ko" indicator

Reply:
I just renamed `NotoSans-Regular.woff` to `noto-sans-korean-regular.woff` and updated the corresponding CSS, I hope this is enough.

Reply:
yep, that'll do it, in it goes!